testgrid/config/config.yaml

```diff
@@ -4834,9 +4834,17 @@ dashboards:
   dashboard_tab:
   - name: crio-e2e-fedora
     test_group_name: test_pull_request_crio_e2e_fedora
+    open_test_template:
+        url: https://gcsweb-ci.svc.ci.openshift.org/gcs/<gcs_prefix>/<test-id>.txt
```

Comment:
I think `<changelist>` should work, not sure about `<test-id>`? cc @michelle192837

Reply:
Yes, this should be `<changelist>`. `<test-id>` is something else. Could also point at `https://storage.googleapis.com/<gcs_prefix>/<changelist>.txt`.